.github/workflows/godot-asset-library.yaml

```diff
@@ -0,0 +1,21 @@
+on:
+  release:
+    types:
+      - published
+
+name: Push to Godot Asset Library
+
+jobs:
+  publish:
+    if: github.event.action == 'released'
+    runs-on: ubuntu-latest
+    name: Push new release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Push to Godot Asset Library
+        uses: deep-entertainment/godot-asset-lib-action@v0.4.0
+        with:
+          username: ${{ secrets.GODOT_ASSET_LIBRARY_USERNAME }}
+          password: ${{ secrets.GODOT_ASSET_LIBRARY_PASSWORD }}
+          assetId: TODO
```

Comment:
The underlying API does support creating a new asset rather than just updating an existing asset, but this GitHub Action (which I quickly read the source of – it doesn't appear to be about to steal our password) requires that it be specified. Fair enough.

Reply:
Presuming it gets approved, it should be [12617](https://godotengine.org/asset-library/asset/12617).

Reply:
As it turns out, that was just the ID for the submission I guess. The real ID is [3095](https://godotengine.org/asset-library/asset/3095). Fixup coming.